.github/workflows/ci.yml

```diff
@@ -15,7 +15,9 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           cache: 'yarn' # cache node modules
-          node-version: lts
+          node-version: latest
```

Comment:
it should stay on `lts`

Reply:
`latest` is 19? and `lts` 18?

Reply:
just "lts" doesn't work, the examples are there: https://github.com/actions/setup-node#supported-version-syntax
I will try "lts/*"